mtp_api/apps/prison/serializers.py

```diff
@@ -0,0 +1,34 @@
+from django.db.models import Max
+
+from rest_framework import serializers
+
+from .models import PrisonerLocation
+
+
+class PrisonerLocationListSerializer(serializers.ListSerializer):
+
+    def get_latest_upload_counter(self):
+        result = PrisonerLocation.objects.all().aggregate(max=Max('upload_counter'))
+        return result['max'] or 0
+
+    def create(self, validated_data):
+        # NOTE: this can cause race conditions and can be fixed but do we care?
+        # Not likely to happen
+        next_upload_counter = self.get_latest_upload_counter() + 1
```

Comment:
Do we need the upload counter? Can we just use the locations that've been uploaded in the last day (or the most recent date we have locations for)?